assignment/scripts/aboutMe.js

```diff
@@ -1,59 +1,91 @@
-
-// 6 - Console log this sentence, adding in the variables you created above: 
+const luckyNumber = 8;
+// 6 - Console log this sentence, adding in the variables you created above:
 // 'My name is (full name), and I think (lucky number) is a winner!'.
 // Refer back to the videos if you need help with this one.
-
+console.log(`My name is ${fullName}, and I think ${luckyNumber} is a winner!`);
 // 7 - Create a variable named `adventurous` and set it to a boolean value (true or false)
-
+let adventures = true;
 // 8 - Create a variable named `food`, and set its value to a string of your favorite food
-
+let food = "Burrito";
 // 9 - Create a variable called `pets` and set it to the value of the number of pets you have
-
+let pets = 1;
 // 10 - Create a variable called `friendsPets` and assign it the value of the number of pets your friend has
-
+let friendsPets = 2;
 // 11 - Add two pets to your `pets` variable
-
+let allPets = friendsPets + pets;
 // 12 - Create a constant variable called `allowedPets` and set it to a number value of your choice
-
-// 13 - Create a conditional: if adventurous is true, console log "Adventures are great!", 
+const allowedPets = 3;
+// 13 - Create a conditional: if adventurous is true, console log "Adventures are great!",
 // if it's not true, console log "How about we stay home?"
-
-
+adventures ? console.log("Adventures are great!") : console.log("How about we stay at home ? ");
 // 14 - Create a compound conditional: if luckyNumber is 2 and adventurous is true,
 // console log "Roll the dice!"
-
-// 15 - Write a conditional that console logs "I can have more pets!" 
+if (luckyNumber == 2 && adventures) {
+  console.log("Roll the dice!");
+}
+// 15 - Write a conditional that console logs "I can have more pets!"
 //if the value of `pets` is less than the value of `allowedPets`,
 // console logs "I have enough pets" if the value of `pets` is equal to the value of `allowedPets`,
 // and console logs "Oh no, I have too many pets!"
 // if the value of `pets` is greater than the value of `allowedPets`.
-
+if (pets < allowedPets) {
+  console.log("I can have more pets!");
+} else if (pets === allowedPets) {
+  console.log("I have enough pets");
+} else {
+  // another condition is not necessary here IMO, since there can be no other option
+  console.log("I have too many pets!");
+}
 // STRETCH GOALS:
 
 // 16 - Make a variable called `mostPets` and a conditional that
 // correctly checks the `pets` and `friendsPets` variables, and
 // assigns the highest value to `mostPets`. There's several possibilities --
-// be sure to think through all the scenarios. 
+// be sure to think through all the scenarios.
 // console.log `mostPets` after the conditional has run.
 
+let mostPets;
+
+if (pets > friendsPets) {
+  mostPets = pets;
+  console.log("I have the most pets", mostPets);
+} else if (pets === friendsPets) {
+  console.log("We have the same amount of pets", mostPets);
```

Comment:
Uh oh! You never assigned a value to `mostPets` in this case, so it will be `undefined`!